backend/eduschola_project/edusch_app/serializers.py

```diff
@@ -1,46 +1,55 @@
 from rest_framework import serializers
-from .models import Student, Parent, Instructor, User, School, Assignment
+
+from django.contrib.auth.models import Group
+from .models import Student, Parent, Staff, User, School, Grade, Assignment
+
 
 
 class UserSerializer(serializers.ModelSerializer):
+    # groups = serializers.StringRelatedField(many=True, read_only=True)
     class Meta:
         model = User
-        fields = '__all__'
-
 
-class StudentSerializer(serializers.ModelSerializer):
-    user = UserSerializer()
+        fields = ['username', 'password', 'role', 'school', 'first_name', 'last_name']
```

Comment:
this password added to this list checks at the point of creating a user and not sending data out? @abrokinla

Reply:
Done__

Reply:
I think this is an old code. This is my updated studentserializer
 class StudentSerializer(serializers.ModelSerializer):
    user = UserSerializer(read_only=True)
    parent = ParentSerializer(read_only=True)
    class Meta:
        model = Student
        fields = ['student_id', 'user', 'date_of_birth', 'phone_number', 'address', 'parent', 'school']